--- a/gcc/cp/except.c	2020-03-11 23:18:24.598231737 +0000
+++ b/gcc/cp/except.c	2020-03-11 23:19:17.438281907 +0000
@@ -1025,7 +1025,8 @@
      unless the system headers are playing rename tricks, and if
      they are, we don't want to be confused by them.  */
   id = DECL_NAME (fn);
-  return !!libc_name_p (IDENTIFIER_POINTER (id), IDENTIFIER_LENGTH (id));
+  return !!libc_name::libc_name_p (IDENTIFIER_POINTER (id),
+		  		IDENTIFIER_LENGTH (id));
 }
 
 /* Returns nonzero if an exception of type FROM will be caught by a
